virttest/qemu_vm.py

```diff
@@ -2142,10 +2152,14 @@ def create_serial_console(self):
         except IndexError:
             raise virt_vm.VMConfigMissingError(self.name, "serial")
         log_name = "serial-%s-%s.log" % (tmp_serial, self.name)
+        if 'ppc' in self.params.get('vm_arch_name', arch.ARCH) and 'tcp'in self.params.get('pro_backend'):
+            create_serial_cmd = "nc %s %s" % (self.params.get('pro_host'), self.params.get('pro_port'))
+        else:
+            create_serial_cmd = "nc -U %s" % self.get_serial_console_filename(tmp_serial)
```

Comment:
I didn't use any tool to check the patch, you can refer to: https://www.python.org/dev/peps/pep-0008/#maximum-line-length
BTW, when you update your patch and commit it according the comments, you can use ''--amend", or you need rebase your branch to squash these new commits into one. For now, there three commits in this PR, and I think they all for one purpose, so you may need rebase.